### PR TITLE
updated cohort 47 organization

### PR DESCRIPTION
**Pull request by brendalong**

### CODE REVIEW

**Comment by lcurtis4** [comment]
👍

File: book-4-nashville-kennels/chapters/AUTHENTICATION.md [hunk 1]
```diff
@@ -9,68 +9,65 @@ In this chapter, you are going to enable two things:
...
 
-Now you can install the mock authentication components into your application.
+Now install the mock authentication components into your application.
 
 1. In the terminal, make sure you are in the `kennels` directory.
 1. Run the following command in that directory.
 
     ```sh
-    /bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/nashville-software-school/client-side-mastery/master/book-4-nashville-kennels/chapters/scripts/auth.sh)"
+    /bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/nss-day-cohort-47/client-side-mastery/master/book-4-nashville-kennels/chapters/scripts/auth.sh)"
```
Comment (askingalot) [inline book-4-nashville-kennels/chapters/AUTHENTICATION.md:70]: this URL will need to be updated

File: book-4-nashville-kennels/chapters/COMPONENT_STYLING.md [hunk 2]
```diff
@@ -0,0 +1,106 @@
+# Modular CSS for React Components
```
Comment (askingalot) [inline book-4-nashville-kennels/chapters/COMPONENT_STYLING.md:1]: Are we using this file?

File: book-4-nashville-kennels/chapters/DOCKER_INTRO.md [hunk 3]
```diff
@@ -0,0 +1,3 @@
+# Introduction to Docker
+
+## Coming Soon
```
Comment (askingalot) [inline book-4-nashville-kennels/chapters/DOCKER_INTRO.md:3]: How soon?

**Review by askingalot — CHANGES_REQUESTED** [review]
Only one real issue. The rest are just comments / questions

**Pull request merged**